### ISSUE TITLE
I can't get manifest

### ISSUE THREAD

**Issue by adem74** [Hexer10/youtube_explode_dart#89]
**Describe the bug**
When I try get audio stream url, I get this error. It was working until these days.
**To Reproduce**
```dart

                                  var yt = youtube.YoutubeExplode();
                                  yt.videos.streamsClient
                                      .getManifest(videoList[index].id)
                                      .then((value) => print(value.toString()));

```

**Stacktrace**
E/flutter (24409): [ERROR:flutter/lib/ui/ui_dart_state.cc(177)] Unhandled Exception: FormatException: Unexpected character (at character 2)
E/flutter (24409): {args:{raw_player_response:ytInitialPlayerResponse}}
E/flutter (24409):  ^

Also all error codes;
https://hizliresim.com/ugsrMf

**Comment by adem74** [comment]
I get some error with example codes of plugin

**Comment by pcevandijk** [comment]
I got the same error.

**Comment by Hexer10** [comment]
Should be fixed with `1.7.3` please open I new issue if this still happens.

**Comment by adem74** [comment]
> Should be fixed with `1.7.3` please open a new issue if this still happens.

Thank you so much

**Comment by nasirjavaid829** [comment]
i'm facing same issue with 1.7.3 ....
